easybuild/easyconfigs/r/resolos/resolos-0.3.5-GCCcore-11.3.0.eb

```
@@ -0,0 +1,38 @@
+easyblock = 'PythonBundle'
+
+name = 'resolos'
+version = '0.3.5'
+
+homepage = 'https://gitlab.unige.ch/resolos/resolos'
+description = """Resolos is a toolkit written in Python for maintaining reproducible environments 
+for scientific computations. It's main goal is to enable researchers to easily
+replicate environments through space (running code on HPC environment)
+and time (environment preservation for long term archival).
+For installation and detailed usage, check out the documentation."""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
+builddependencies = [('binutils', '2.38')]
+
+dependencies = {
+    ('Python', '3.10.4'),
+    ('PyYAML', '6.0'),
+}
+
+use_pip = True
+
+exts_list = [
```

Thanks for the update. The installation now fails because the sanity checks with `pip` detected missing dependencies. The modules `click-log`, `conda-pack`, `semver` and `PyJWT` are [`install_requires` of resolos](https://gitlab.unige.ch/resolos/resolos/-/blob/main/setup.py#L29) but are not provided by either the [Python v3.10.4 easyconfig](https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/p/Python/Python-3.10.4-GCCcore-11.3.0.eb), PyYAML v6.0 or any other easyconfig in our repo. Therefore, we should add them as additional extensions to this easyconfig as follows:
```suggestion
exts_list = [
    ('click-log', '0.4.0', {
        'checksums': ['3970f8570ac54491237bcdb3d8ab5e3eef6c057df29f8c3d1151a51a9c23b975'],
    }),
    ('conda-pack', '0.6.0', {
        'checksums': ['615f41353fdc8793b0edb55702212273f17e40b43cc2c17493430e9c66d9f0b3'],
    }),
    ('semver', '2.13.0', {
        'checksums': ['fa0fe2722ee1c3f57eac478820c3a5ae2f624af8264cbdf9000c980ff7f75e3f'],
    }),
    ('PyJWT', '2.6.0', {
        'modulename': 'jwt',
        'checksums': ['69285c7e31fc44f68a1feb309e948e0df53259d579295e6cfe2b1792329f05fd'],
    }),
```